packages/schema/lib/schemas/FieldSchema.js

```diff
@@ -19,192 +13,132 @@ const formatBullet = (f) => `* ${f.map(wrapInBackticks).join(' & ')}`;
-        type: 'boolean',
-      },
-      inputFormat: {
-        description:
-          'Useful when you expect the input to be part of a longer string. Put "{{input}}" in place of the user\'s input (IE: "https://{{input}}.yourdomain.com").',
-        type: 'string',
-        // TODO: Check if it contains one and ONLY ONE '{{input}}'
-        pattern: '^.*{{input}}.*$',
-      },
-      meta: {
-        description:
-          'Allows for additional metadata to be stored on the field. Supports simple key-values only (no sub-objects or arrays).',
-        $ref: FieldMetaSchema.id,
-      },
-    },
-    examples: [
-      { key: 'abc' },
-      { key: 'abc', choices: { mobile: 'Mobile Phone' } },
-      { key: 'abc', choices: ['first', 'second', 'third'] },
-      {
-        key: 'abc',
-        choices: [{ label: 'Red', sample: '#f00', value: '#f00' }],
-      },
-      { key: 'abc', children: [{ key: 'abc' }] },
-      { key: 'abc', type: 'integer', helpText: 'neat' },
-      {
-        key: 'abc',
-        type: 'integer',
-        meta: {
-          internalType: 'numeric',
-          should_call_api: true,
-          display_order: 1,
-        },
-      },
-    ],
-    antiExamples: [
-      {
-        example: {},
-        reason: 'Missing required key: key',
-      },
-      {
-        example: { key: 'abc', type: 'loltype' },
-        reason: 'Invalid value for key: type',
-      },
-      {
-        example: { key: 'abc', choices: {} },
-        reason: 'Invalid value for key: choices (cannot be empty)',
-      },
-      {
-        example: { key: 'abc', choices: [] },
-        reason: 'Invalid value for key: choices (cannot be empty)',
-      },
-      {
-        example: { key: 'abc', choices: [3] },
-        reason:
-          'Invalid value for key: choices (if an array, must be of either string or FieldChoiceWithLabelSchema)',
-      },
-      {
-        example: { key: 'abc', choices: [{ label: 'Red', value: '#f00' }] },
-        reason:
-          'Invalid value for key: choices (if an array of FieldChoiceWithLabelSchema, must provide key `sample`)',
-      },
-      {
-        example: { key: 'abc', choices: 'mobile' },
-        reason:
-          'Invalid value for key: choices (must be either object or array)',
-      },
-
-      {
-        example: { key: 'abc', children: ['$func$2$f$'] },
-        reason:
-          'Invalid value for key: children (must be array of FieldSchema)',
-      },
-    ],
-    additionalProperties: false,
+module.exports = makeSchema({
+  id: '/FieldSchema',
+  description: `Defines a field an app either needs as input, or gives as output. In addition to the requirements below, the following keys are mutually exclusive:\n\n${incompatibleFieldsList}`,
```

Comment:
```suggestion
  description: `In addition to the requirements below, the following keys are mutually exclusive:\n\n${incompatibleFieldsList}`,
```

Intended to be appended to the description of each of the schemas inheriting from FieldSchema.